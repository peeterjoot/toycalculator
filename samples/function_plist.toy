FUNCTION bar ( INT16 w, INT32 z )
{
    PRINT "In bar";
    PRINT w;
    RETURN;
};

FUNCTION foo ( )
{
    INT16 v;
    v = 3;
    PRINT "In foo";
    CALL bar( v, 42 );
    PRINT "Called bar";
    RETURN;
};

PRINT "In main";
CALL foo();
PRINT "Back in main";
