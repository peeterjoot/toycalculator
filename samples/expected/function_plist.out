In main
In foo
In bar
3
42
Called bar
Back in main
